src/vs/workbench/contrib/chat/browser/chatEditing/notebook/chatEditingNotebookEditorIntegration.ts

```diff
@@ -72,9 +74,11 @@ export class ChatEditingNotebookEditorIntegration extends Disposable implements
 		this.integration.enableAccessibleDiffView();
 	}
 	acceptNearestChange(change: IModifiedFileEntryChangeHunk): void {
+		this._accessibilitySignalService.playSignal(AccessibilitySignal.keepEdits, { allowManyInParallel: true });
```

Comment:
I let @DonJayamanne comment but I guess it is similar to my comment in the code editor integration and that the signal should be played from the place that "does it"